src/components/ViewTask.js

```diff
@@ -1,51 +1,122 @@
 /** @jsx jsx */
 import { jsx } from 'theme-ui';
 import PropTypes from 'prop-types';
-import { AiTwotoneCalendar } from 'react-icons/ai';
+import { GrCalendar } from 'react-icons/gr';
+import { GrNotes } from 'react-icons/gr';
+import { GrTask } from 'react-icons/gr';
 
 function ViewTask({ task }) {
-  const h3Div = {
-    display: 'flex',
-  };
-  const AiTwoTone = {
-    paddingTop: '25px',
-  };
-  const dueDate = {
-    paddingLeft: '20px',
-    color: '#b6b6b6',
-  };
-  const dueDateActual = {
-    paddingLeft: '40px',
-    marginTop: '5px',
-  };
-  const categoryName = {
-    paddingLeft: '35px',
-  };
-
   return (
     <div>
-      <h1>{task.task_name}</h1>
-      <div sx={h3Div}>
-        <div sx={AiTwoTone}>
-          <AiTwotoneCalendar />
+      <h1
```

Comment:
Probably shouldn't be an `h1` since this is a modal over the top of all of the other html on the page (and you should only have one `h1` tag on a page)